docs/process/release.md

```diff
@@ -159,6 +149,12 @@ using the `--githubToken` flag. You just then have to confirm the draft.
 
 > **Tags containing `next` or `rc` should be marked as pre-release.**
 
+## Post-release Version Update
```

Comment:
Do we want to document/expect any particular commit message? Release commits are very particular, though it is probably less important here.

Reply:
Added commit message.
```sh
git commit package.json packages/schematics/angular/utility/latest-versions.ts -m "build: bump version to vXX"
```